proto/teleport/lib/teleterm/v1/database.proto

```diff
@@ -42,4 +43,14 @@ message Database {
   string addr = 7;
   // labels is a list of labels for this database
   repeated Label labels = 8;
+  // target_health of the "db_server" that is serving this database.
+  TargetHealth target_health = 9;
+}
+
+// DatabaseServer describes a database server resource (resource kind "db_server")
+message DatabaseServer {
+  string cluster_uri = 1;
```

Comment:
i wasn't sure if i needed to include `cluster_uri`, when currently, i don't use it, but it seems to be the custom so I added it

Reply:
This should be just `uri`.